### PR TITLE
Fix: onboarding profile bio textarea length

### LINKED ISSUE #17863: Setting short bio when logging in first time
**Describe the bug**

When I logged in first time to forem I’ve inserted a “short” bio, longer than 200 characters. There wasn’t any error message, I just couldn’t go to the next page. I know the length limitation 'cause I was able to fill it in the settings, where it works properly.

I was logged in with Github account.

**To Reproduce**

**Expected behavior**

Showing something to be able to find out, what is the problem. Or a counter, like in the settings, that how many characters I have.

**Screenshots**

**Desktop (please complete the following information):**

- OS, version: Fedora
- Browser, version: Chrome

**Smartphone (please complete the following information):**

- Device:
- OS, version:
- Browser, version:

**Additional context**

### PR DESCRIPTION
**Pull request by jeferson-sb**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Add maxlength to textarea and username fields at the onboarding flow so the user can perceive the amount of characters first and avoid seeing an error message coming from the backend validation.

## Related Tickets & Documents

- Related Issue #17863
- Closes #17863

## Added/updated tests?

- [ ] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?
No

### CODE REVIEW

File: app/javascript/onboarding/components/ProfileForm.jsx [hunk 1]
```diff
@@ -230,9 +233,21 @@ export class ProfileForm extends Component {
...
                   required: false,
+                  maxLength: SUMMARY_MAXLENGTH,
                 }}
                 onFieldChange={this.handleFieldChange}
               />
+              <p
+                id="summary-description"
+                class="crayons-field__description align-right"
+              >
+                <span class="screen-reader-only" aria-live="polite">
```
Comment (fdocr) [inline app/javascript/onboarding/components/ProfileForm.jsx:244]: Love the use of `aria-live="polite"` within the `screen-ready-only` section here 👏🏼

**Review by fdocr — APPROVED** [review]
This is a great contribution from where I see it. Thank you @jeferson-sb

**Comment by jeferson-sb** [comment]
@jaw6 Could you review?

**Review by jaw6 — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**